Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine as build
+
+RUN apk update && \
+  apk add curl bash && \
+  curl -sfL https://install.goreleaser.com/github.com/caarlos0/svu.sh | bash -s -- -b /usr/local/bin
+
+FROM scratch
+
+COPY --from=build /usr/local/bin/svu /usr/local/bin/svu
```

Comment:
this can be just

```suggestion
FROM scratch

COPY svu /usr/local/bin/svu
```

Reply:
Updated. Should we still keep the `ENTRYPOINT` there?